EIPS/eip-4626.md

```diff
@@ -1,7 +1,7 @@
 ---
 eip: 4626
-title: Tokenized Vault Standard
-description: A standard for tokenized Vaults with a single underlying ERC-20 token.
+title: Tokenized Vaults
+description: Tokenized Vaults with a single underlying EIP-20 token.
```

Comment:
@Pandapip1 @MicahZoltu this is okay to update title like this after Finalization?

Reply:
Yes